src/components/Select/NativeSelect.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <div class="native-select" :class="classes.select">
+    <div class="native-select__input-container">
+      <select ref="input" v-model="selected" class="native-select__input" v-bind="{ disabled, required }">
```

Comment:
didn't you recently comment on one of my PRs about avoiding v-bind with non-dynamic props?